cmd/oci-image-tool/man/oci-image-tool-unpack.1.md

```diff
@@ -0,0 +1,50 @@
+% OCI(1) OCI-IMAGE-TOOL User Manuals
+% OCI Community
+% JULY 2016
+# NAME
+oci-image-tool-unpack \- Unpack an image or image source layout
+
+# SYNOPSIS
+**oci-image-tool unpack** [src] [dest] [flags]
+
+# DESCRIPTION
+`oci-image-tool unpack` unpacks a given OCI image into a directory suitable to be used with `runc`.
```

Comment:
On Fri, Jul 22, 2016 at 10:25:35AM -0700, Antonio Murdaca wrote:
> > +# DESCRIPTION
> > +`oci-image-tool unpack` unpacks a given OCI image into a directory suitable to be used with `runc`.
> 
> sure, any suggestion on the language?

For unpack, something like:

  Validates an application/vnd.oci.image.manifest.v1+json and unpacks
  its layered filesystem to DEST.

For create-runtime-bundle, something like:

  Validates an application/vnd.oci.image.manifest.v1+json and unpacks
  its layered filesystem to DEST/rootfs, although the target directory
  is configurable with --rootfs.  See oci-image-tool-unpack(1) for
  more details on this process.

  Also translates the referenced config from
  application/vnd.oci.image.serialization.config.v1+json to a
  runtime-spec's config.json.